.cruft.json

```diff
@@ -0,0 +1,37 @@
+{
+  "template": "/home/mcus/repos/cookiecutter_agent_plugin_builder/../infection-monkey-cookiecutter",
```

Comment:
This should point to the repository in github, otherwise no one but @ilija-lazoroski can ever update cruft.